packages/meditrak-app/app/assessment/selectors.js

```diff
@@ -3,8 +3,17 @@
  * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
  **/
 
+import { getVariables, runArithmetic } from '@beyondessential/arithmetic';
 import { checkAnswerPreconditionsAreMet } from './helpers';
 
+const CONDITION_TYPE = {
+  '=': (value, filterValue) => value === filterValue,
+  '>': (value, filterValue) => value > filterValue,
+  '<': (value, filterValue) => value < filterValue,
+  '>=': (value, filterValue) => value >= filterValue,
+  '<=': (value, filterValue) => value <= filterValue,
+};
```

Comment:
@kael89 do you know where the `analyticCount` indicator development is up to? We were thinking of having a common set of comparison functions that parsed a string and returned a boolean result, in some published package right?

Reply:
That's correct, we also thought of using math.js for empowering that behaviour in indicators. Options:

A. Publish a package that exposes a common set of comparison options with custom code
B. Publish a package that exposes a common set of comparison options, wrapping math.js
C. Do not publish a package but use math.js directly

Selecting either A, B or C will most likely depend on the quality and functionality of math.js (which I am unaware of)

And then
i. Refactor data builders to use either the published package/math.js directly, or
ii. Leave this to be done in the reporting package work

Reply:
@chris-bes could you take a look at math.js and see what you think? With the aim of dropping it into this PR wherever @tupaia/arithmetic is currently used, as well as where the condition checking is done